File: yt/yt/flow/examples/java/noop/noop/src/main/java/tech/ytsaurus/flow/examples/noop/PipelineMain.java
package tech.ytsaurus.flow.examples.noop;

import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.boot.builder.SpringApplicationBuilder;

@SpringBootApplication
public class PipelineMain {

    private PipelineMain() {
    }

    public static void main(String[] args) throws Exception {
        new SpringApplicationBuilder(PipelineMain.class).run(args);
    }
}

File: yt/yt/flow/examples/kotlin/wait_click_join/wait_click_join/src/main/kotlin/tech/ytsaurus/flow/examples/waitclickjoin/PipelineMain.kt
package tech.ytsaurus.flow.examples.waitclickjoin

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.builder.SpringApplicationBuilder

@SpringBootApplication
open class PipelineMain {
    companion object {
        // [BEGIN main]
        @JvmStatic
        fun main(args: Array<String>) {
            SpringApplicationBuilder(PipelineMain::class.java)
                .run(*args)
        }
        // [END main]
    }
}
